proxy/config.go

```diff
@@ -128,3 +130,25 @@ func AddService(filepath string, service Service) error {
 	_, err := file.WriteString(serviceStr)
 	return err
 }
+
+// RemoveService remves a service from the filepath
+func RemoveService(filepath string, service Service) error {
+	file, err := ioutil.ReadFile(filepath)
+
+	if err != nil {
+		log.Println("File error: %v\n", err)
+		return err
+	}
+
+	serviceStr := service.Name + " " + service.URL + "\n"
```

Comment:
When loading services the regex used is (\S+). This allows for several spaces between _name_ and _url_. You should search by regex or make a line by line search and search for _name_ and then _url_. Otherwise you would fail to remove something like ```test          http://localhost:9999``` and you would remove it only if it is ```test http://localhost:9999```